spec/integration/encryption_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+require 'support/crypt/models'
+
+describe 'Encryption' do
+  require_enterprise
```

Comment:
It appears community edition server supports explicitly-declared encryption (i.e. not auto/schema map style)
https://www.mongodb.com/docs/manual/core/csfle/

Reply:
Thank you for your clarification.

One more question: how exactly are schema maps applied? Does the user need to run a migration to apply the latest schema map to the DB? What if two different clients are using two different JSON schema map definitions with accessing the same database?

> What would be your use case where you need to be able to control everything from Rails?

I will need to see if there are any limitations with the auto-encryption approach when it is ready to test. For example, whether it allows me to encrypt existing unencrypted data seamlessly.